roles/vernemq/templates/z_vernemq_cluster.yaml.j2

```diff
@@ -45,6 +45,30 @@ spec:
       port: 1888
       websocket: true
     plugins: []
+  sidecarBaseImage: dojot/cert-sidecar
+  sidecarEnv:
+    - name: CERT_SC_APP_SIDECAR_TO
+      value: "vernemq-k8s"
+    - name: CERT_SC_LOG_CONSOLE_LEVEL
+      value: "info"
+    - name: CERT_SC_CERTS_HOSTNAMES
+      value: "[\"vernemq-k8s\", \"localhost\"]"
```

Comment:
Change `localhost` to `{{ dojot_domain_name }}`

Reply:
Changed!